### PR TITLE
add Bloxroute sdk

### PR DESCRIPTION
**Pull request by tiennampham23**
## 📝 Summary
Use Bloxroute SDK to listen to new pending transactions from the Bloxroute data feed

## ⛱ Motivation and Context
Separate Bloxroute and Eden into two components, with this [PR](https://github.com/flashbots/mempool-dumpster/pull/23).

## 📚 References

---

## ✅ I have run these commands

* [x] `make lint`
* [x] `make test`
* [x] `go mod tidy`

### CODE REVIEW

File: collector/collector.go [hunk 1]
```diff
@@ -41,20 +40,14 @@ func Start(opts *CollectorOpts) {
...
-		}
+		blxConn := NewBlxNodeConnection(blxOpts, processor.txC)
+		go blxConn.Start()
 	}
 
 	if opts.EdenAuthToken != "" {
 		blxOpts := BlxNodeOpts{ //nolint:exhaustruct
 			Log:        opts.Log,
 			AuthHeader: opts.EdenAuthToken,
-			IsEden:     true,
```
Comment (tiennampham23) [inline collector/collector.go:57]: This line will be replaced by PR #23

File: scripts/subscibe-test/main.go [hunk 2]
```diff
@@ -59,7 +59,6 @@ func MainEden() {
 		Log:        log,
 		AuthHeader: os.Getenv("EDEN_AUTH_HEADER"),
 		URL:        "wss://speed-eu-west.edennetwork.io",
-		IsEden:     true,
```
Comment (tiennampham23) [inline scripts/subscibe-test/main.go:62]: Same as above, this line will be replaced by PR #23

**Review by tiennampham23 — COMMENTED** [review]
Comment for some confusing lines about the Eden network connection

**Comment by tiennampham23** [comment]
@metachris Please take a look

**Comment by metachris** [comment]
thanks! will get to it soon

**Comment by metachris** [comment]
thanks, but not needed anymore, current bloxroute connection was contributed by bloxroute team and works well

**Comment by tiennampham23** [comment]
@metachris why? BloxRoute created an SDK repository that exposes the API to other services. You still need to manage their logic and it will be broken in the future? 

Haha, I don't think it is a good rule for open source repositories but this repository is from you. So thanks

**Comment by metachris** [comment]
The current bloXroute node connector is written and maintained by the bloXroute team, and it's good for them to have ownership. Feel free to talk to them about changing the implementation.

Nevertheless, thanks for the effort 🙌

**Pull request closed without merging**